infra/nginx.conf

```diff
@@ -13,11 +12,15 @@ server {
         # Introduced by https://github.com/gratipay/grtp.co/issues/144
         ssl_dhparam dhparams.pem;
 
-        return 302 $scheme://grtp.co$request_uri;
+        return 301 https://grtp.co$request_uri;
```

Comment:
301s give me the willies, because they're hard to shake from caches once used (I know we shouldn't ever _need_ to, but ...). Can we stick with 302?